### DISCUSSION TITLE
None of the recent versions have ksu?

### DISCUSSION THREAD

**Discussion by youlan2333** [MustardChef/WSABuilds#326] [General]
None of the recent versions have ksu?

**Comment by youlan2333** [comment]
Sorry, I didn't see it :)

**Reply by MustardChef** [reply]
Yeah, the ksu version for the latest build was added to another release due to a bug in the GitHub Actions Script.

**Reply by youlan2333** [reply]
>  Yeah, the ksu version for the latest build was added to another release due to a bug in the GitHub Actions Script.

I don't know why, but after upgrading from Win10 to Win11, I uninstalled and reinstalled WSA but couldn't install it anymore. T^T
